### PR TITLE
updating the readme file [finishes #170944088]

### PR DESCRIPTION
**Pull request by anasey01**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,78 +1,136 @@
...
+3. Your connection should be successful
+
+HAPPY CODING!!!
+
+Common docker commands
+ - Use `docker-compose start` command to restart all containers.
+ - Use `docker-compose stop` command to stop all containers.
+ - Use `docker start <container id>` to start a specific container.
+ - Use `docker stop <container id>` to stop a specific container.
+ - Use `docker images command to view all docker images.
```
[the remark this answers is not in the document]
Comment (anasey01) [reply README.md]: Good catch

File: README.md [hunk 2]
```diff
@@ -1,78 +1,136 @@
...
+ - Use `docker ps or docker ps -a` to view all containers. The -a flag means stopped containers are listed as well.
+ - Use `docker system prune -a to` remove all stopped containers and images at once.
+ - Use `docker rmi <image id> `or `docker rmi <image name>:<tag>` to remove a specific image.
+ - Use `docker rm <container id>` to remove a specific stopped container.
+ - Use `docker-compose exec <service-name> <command>` command to execute commands in a container, where service-name is the docker service (for an image/container) specified in the docker-compose.yml file. For example to execute the artisan command to clear laravel config cache for the php-fpm service we run docker-compose exec php-fpm php artisan config:clear. Other artisan commands are run similarly against the docker container containing our laravel app.
```
[the remark this answers is not in the document]
Comment (anasey01) [reply README.md]: Tried this but then cause sentence is a continuation of the preceding statement, adding a new line suggested it to be a new paragraph.

[the remark this answers is not in the document]
Comment (anasey01) [reply README.md]: Screenshot 2020-01-31 at 14 09 36

[the remark this answers is not in the document]
Comment (Frediflexta) [reply README.md]: Okay, you can leave it as is if you wonna, but it's more readable as seen on this image you just shared.

**Pull request merged**